practices/kafka.md

```diff
@@ -12,27 +12,34 @@ tags:
   - integration
   - devops
   - stream processing
+  - streaming
 keywords:
   - kafka
   - deployment
   - messaging
   - message bus
   - stream processing
+  - streaming
 covered_by_dx_scanner: false
 ---
 
 # Apache Kafka
 
 **TL;DR**
 
-aaa
+Apache Kafka is an event streaming platform initially used as messaging queue created as open-source by LinkedIn in 2011.
```

Comment:
I would suggest restructuring this sentence a bit. Using `as` twice in one sentence does not sound very native to me. My rough suggestion:

Apache Kafka is an open-source event streaming platform created by LinkedIn in 2011 which initially served as a messaging queue.

Reply:
Thanks!